lsst/ts/laser/component.py

```diff
@@ -1,8 +1,10 @@
-"""Implements component classes.
+"""Implements the component class for the TunableLaser.
 
 """
-import serial
-from types import SimpleNamespace
+import logging
+from lsst.ts.laser.hardware import *
```

Comment:
Some more `import *` that I suggest you change. I bet `flake8` will complain though.